neptune-export/src/main/java/com/amazonaws/services/neptune/propertygraph/schema/DataType.java

```diff
@@ -377,8 +382,11 @@ public static String escapeSeparators(Object value, String separator) {
         return temp.replace(separator, "\\" + separator);
     }
 
-    public static String escapeDoubleQuotes(Object value) {
-        String temp = value.toString().replace("\"\"", "\"");
+    public static String escapeDoubleQuotes(Object value, boolean isRewrite) {
+        String temp = value.toString();
+        if (isRewrite) {
+            temp = temp.replace("\"\"", "\"");
+        }
         return temp.replace("\"", "\"\"");
```

Comment:
Could change this to
```
        if (!isRewrite) {
            temp = temp.replace("\"", "\"\"");
        }
        return temp;
```
to avoid singling then doubling quotes

Reply:
Ah yes, I think I was trying to just use the boolean as is, but this would be better.